README.md

```diff
@@ -129,6 +129,14 @@ and complete authentication of the user.
 
 [the specification]: https://tools.ietf.org/html/rfc6749#section-2.3.1
 
+### PKCE
+
+Some oauth providers require an additional step called Proof Key for 
+Code Exchange 
+([PKCE](https://www.oauth.com/oauth2-servers/pkce/authorization-request/)).  Ring-OAuth2
+will include a proof key in the workflow when `:pkce? true`is included 
+in the configuration map.
```

Comment:
Let's write it as:

```markdown
Some OAuth providers require an additional step called *Proof Key for
Code Exchange* ([PKCE][]). Ring-OAuth2 will include a proof key in the
workflow when `pkce?` is set to `true` in the profile map.

[pkce]: https://www.oauth.com/oauth2-servers/pkce/authorization-request/
```

This is a little neater, is within line limits, and uses the same terminology as the rest of the README.

Reply:
oh nice, I didn't realize README had this feature